lnd/setup/index.js

```diff
@@ -68,24 +68,20 @@ async function channels(node) {
       await lightning.addPeer(node, p);
     });
 
-    const chans = await lightning.listChannels(node);
-    const channels = chans.channels || [];
-    if (!channels.length) {
```

Comment:
this `if` would not let bob make a channel because alice already made one to bob

Reply:
oh wait now it makes more and more channel every time you start them, fix coming